scripts/package-artifacts.ps1

```diff
@@ -22,23 +22,27 @@
-    Copy-Item -Path "$PSScriptRoot\..\src\SonarScanner.MSBuild\bin\Release\$sourcetfm\SonarScanner.MSBuild.Shim.dll" -Destination "$fullBuildOutputDir\sonarscanner-msbuild-$targettfm"
-    # Take files from netcoreapp3.1 because they are not present in netcoreapp2.1 bin directory
-    Copy-Item -Path "$PSScriptRoot\..\src\SonarScanner.MSBuild\bin\Release\netcoreapp3.1\Google.Protobuf.dll" -Destination "$fullBuildOutputDir\sonarscanner-msbuild-$targettfm"
-    Copy-Item -Path "$PSScriptRoot\..\src\SonarScanner.MSBuild\bin\Release\netcoreapp3.1\Newtonsoft.Json.dll" -Destination "$fullBuildOutputDir\sonarscanner-msbuild-$targettfm"
-    Copy-Item -Path "$PSScriptRoot\..\src\SonarScanner.MSBuild.Tasks\bin\Release\netstandard2.0\SonarScanner.MSBuild.Tasks.dll" -Destination "$fullBuildOutputDir\sonarscanner-msbuild-$targettfm"
-    [System.IO.Compression.ZipFile]::ExtractToDirectory("$scannerCliDownloadDir\$scannerCliArtifact", "$fullBuildOutputDir\sonarscanner-msbuild-$targettfm")
-    Compress-Archive -Path $fullBuildOutputDir\sonarscanner-msbuild-$targettfm\* -DestinationPath $fullBuildOutputDir\sonarscanner-msbuild-$targettfm.zip -Force
+function Package-NetScanner() {
+    $sourcetfm = "netcoreapp3.1"
+    $sourceRoot = "$PSScriptRoot\..\src\SonarScanner.MSBuild\bin\Release\$sourcetfm"
+    $destination = "$fullBuildOutputDir\sonarscanner-net"
+    $destinationTargets = "$destination\Targets"
+
+    if (!(Test-Path -path "$destination")) {New-Item "$destination" -Type Directory}
+    if (!(Test-Path -path "$destinationTargets")) {New-Item "$destinationTargets" -Type Directory}
+    Copy-Item -Path "$PSScriptRoot\..\src\SonarScanner.MSBuild.Tasks\Targets\*" -Destination "$destinationTargets" -Recurse
+    Copy-Item -Path "$sourceRoot\SonarQube.Analysis.xml" -Destination "$destination"
+    Copy-Item -Path "$sourceRoot\SonarScanner.MSBuild.Common.dll" -Destination "$destination"
+    Copy-Item -Path "$sourceRoot\SonarScanner.MSBuild.dll" -Destination "$destination"
+    Copy-Item -Path "$sourceRoot\SonarScanner.MSBuild.PostProcessor.dll" -Destination "$destination"
+    Copy-Item -Path "$sourceRoot\SonarScanner.MSBuild.PreProcessor.dll" -Destination "$destination"
+    Copy-Item -Path "$sourceRoot\SonarScanner.MSBuild.runtimeconfig.json" -Destination "$destination"
+    Copy-Item -Path "$sourceRoot\SonarScanner.MSBuild.Shim.dll" -Destination "$destination"
+    Copy-Item -Path "$sourceRoot\Google.Protobuf.dll" -Destination "$destination"
+    Copy-Item -Path "$sourceRoot\Newtonsoft.Json.dll" -Destination "$destination"
+    Copy-Item -Path "$PSScriptRoot\..\src\SonarScanner.MSBuild.Tasks\bin\Release\netstandard2.0\SonarScanner.MSBuild.Tasks.dll" -Destination "$destination"
+    [System.IO.Compression.ZipFile]::ExtractToDirectory("$scannerCliDownloadDir\$scannerCliArtifact", "$destination")
+    Compress-Archive -Path "$destination\*" -DestinationPath "$destination.zip" -Force
```

Comment:
Educational: do you know why we use to expand the archive via a .NET API (namely `System.IO.Compression.ZipFile`), instead of using the PowerShell idiomatic [`Expand-Archive`](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.archive/expand-archive?view=powershell-7.4)?
Yet, we compress the `$destination.zip` archive using `Compress-Archive`.

Reply:
No, not really. I would make this change in a separate PR through.